storage/src/persistent/database.rs

```diff
@@ -123,6 +123,22 @@ pub trait KeyValueStoreWithSchema<S: KeyValueSchema> {
     fn get_mem_use_stats(&self) -> Result<RocksDBStats, DBError>;
 }
 
+pub trait GetInMemStats {
```

Comment:
@bkontur FYI this had to be separated to it's own trait because of some weird error in the type inferencer when the `get_mem_use_stats` method was called:

```
cannot infer type for type parameter `S` declared on the trait `KeyValueStoreWithSchema`
```

I wouldn't expect the inferencer to need that being that there are no parameters that relate to `S` in that method, but it seems it does.

Reply:
@tizoc 
ok, lets get it working as it is, and we can next week clean this up,
just please do some jira with TODOs for that

Reply:
@bkontur ok, will do that👍

Reply:
This is probably also related to the other duplication issue I mentioned, we have two traits that look very similar.

Reply:
https://simplestakingcom.atlassian.net/browse/TE-432